virtual-setup/rstudio-login.md

```diff
@@ -1 +1,55 @@
-## RStudio Server instructions
\ No newline at end of file
+## RStudio Server Set Up
+
+We will send you your Username and temporary password by a direct message on Slack, after you are logged in on [Cancer Data Science](http://ccdatalab.org/slack).
+
+Go to https://rstudio.ccdatalab.org (You may want to bookmark this) and type in your username and temporary password, and click `Sign in`.
+
+![RStudio Login](screenshots/rstudio-server-login.png)
+
+Signing in should bring you to the RStudio session page.
+Click on the `Terminal` tab.
+
+![RStudio Session](screenshots/rstudio-session-terminal.png)
+
+Type in the `passwd` command in the `Terminal` tab.
+
+![RStudio change password](screenshots/rstudio-password-change-1.png)
+
+...and press `Enter`. Then type in the password you were given in Slack.
```

Comment:
```suggestion
Press `Enter`. Then type in the password you were given in Slack.
```

This line gets a little lost for me in between the screenshots. That may be helped by making the adjacent screenshots smaller or playing around with `<br>`.